docbook/reference/en/en-US/modules/RESTEasy_Client_Framework.xml

```diff
@@ -795,10 +795,18 @@ response.close();
      </para>
 
     <para>
-        Finally, if your javax.ws.rs.client.Client class has created the engine automatically for you, you should
+        If your javax.ws.rs.client.Client class has created the engine automatically for you, you should
         call Client.close() and this will clean up any socket connections.
     </para>
 
+    <para>
+        Finally, given having explicit <methodname>finalize()</methodname> methods can badly affect performances, the 
+        <classname>org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine</classname> flavour
+        of <classname>org.jboss.resteasy.client.jaxrs.engines.ApacheHttpClient43Engine</classname> can be used. With
+        that the user is aways responsible for calling <methodname>close()</methodname> as not <methodname>finalize()</methodname>

```

Comment:
typo "aways"

Reply:
will fix, thanks... I'll also fix the other typo in the same line ;-)

Reply:
Done